build.gradle

```diff
@@ -39,5 +52,18 @@ allprojects {
       events "skipped", "failed", "passed"
     }
   }
-}
 
+  dependencies {
+    errorproneJavac 'com.google.errorprone:javac:9+181-r4173-1'
+    errorprone 'com.google.errorprone:error_prone_core:2.3.2'
+    errorprone 'com.uber.nullaway:nullaway:0.6.4'
+  }
+
+  tasks.withType(JavaCompile).configureEach {
+    options.errorprone {
+      disableWarningsInGeneratedCode = true
+      check("JavaxInjectOnAbstractMethod", CheckSeverity.OFF)
+      option("NullAway:AnnotatedPackages", "dagger.reflect")
```

Comment:
reflect/src/main/java/dagger/Dagger.java
codegen/src/main/java/dagger/Dagger.java
codegen/src/main/java/dagger/internal/DaggerCodegen.java

Reply:
Those packages aren't owned by us so I don't want to include them and give a false sense of security